ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -2287,7 +2307,19 @@ screen and (min-width: 96em) {
 }
 
 .ontario-border-highlight--green {
-  border-color: #2b8737 !important
+  border-color: #2b8737 !important;
+}
+
+.ontario-border-highlight--teal {
+  border-color: #367a76 !important;
+}
+
+.ontario-border-highlight--taupe {
+  border-color: #7b725c !important;
+}
+
+.ontario-border-highlight--red {
+  border-color: #D81A21 !important;
 }
```

Comment:
Can you add a comment just above this rule to say it's specific to our theme? We plan on removing all design system code in favour for their npm package at some point, so we won't to lose this rule